raiden/transfer/channel.py

```diff
@@ -1272,6 +1394,7 @@ def send_lockedtransfer(
     lock = transfer.lock
     channel_state.our_state.balance_proof = transfer.balance_proof
     channel_state.our_state.merkletree = merkletree
+    channel_state.our_state.nonce = transfer.balance_proof.nonce
```

Comment:
This makes the update in `create` unnecessary.

Reply:
There's no update. `create_sendlockedtransfer` only generates a new nonce but doesn't set it to the state. We only set the nonce here.